### PR TITLE
Use syn 2

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ documentation = "https://docs.rs/display_json"
 proc-macro = true
 
 [dependencies]
-syn = { version = "^1.0.0", features = ["derive", "printing", "extra-traits", "parsing"] }
+syn = { version = "^2.0", features = ["derive", "printing", "extra-traits", "parsing"] }
```
Comment (jofas) [inline Cargo.toml]: While we are at it, let's remove the superfluous caret as well :slightly_smiling_face::

```suggestion
syn = { version = "2", features = ["derive", "printing", "extra-traits", "parsing"] }
```

**Review by jofas — COMMENTED** [review]
Thanks for the PR and keeping this crate updated!

File: Cargo.toml (hunk 1 above)
Comment (jayvdb) [reply Cargo.toml]: done

**Comment by jofas** [comment]
Cheers!

**Pull request merged**